### DISCUSSION TITLE
Is this an app or a plugin?

### DISCUSSION THREAD

**Discussion by Neodevils** [AmrDeveloper/CodeView#38] [Q&A] [answered]
Hello, I am new to Android app development. I want to make my own editor which most of people here for this. 

###### If it is an app, where can I download it?

I checked releases but couldn't find 
.apk extension file...

**Comment by AmrDeveloper** [comment, marked as answer]
Hello, @Neodevils,

CodeView is an Android Library (Plugin) that helps you to create a code editor easily for any syntax, the repository contains an example app that you can play with it and see how some features work together,

So if you have Android Studio installed try to clone the repository and run the app module to see the samples then in your app you can import the library using Gradle and create your own app

I am planning to add apk for next releases to be easy to check the sample without Android Studio IDE

**Reply by Neodevils** [reply]
I'll be appreciated. Cause, I am using my phone to code mostly. Thank you ☺️.
